man/rgbgfx.1

```diff
@@ -164,6 +165,37 @@ for a list of formats and their descriptions.
 .It Fl d Ar depth , Fl \-depth Ar depth
 Set the bit depth of the output tile data, in bits per pixel (bpp), either 1 or 2 (the default).
 This changes how tile data is output, and the maximum number of colors per palette (2 and 4 respectively).
+.It Fl i Ar input_tiles , Fl \-input-tileset Ar input_tiles
+Read tiles that will be used to convert this image, and that will always be given the first IDs.
```

Comment:
```suggestion
Use the specified when creating a tile map for the input image, if use along with the
.Fl t
option.
```

Reply:
No, this also affects `-o`.

Reply:
"The first IDs" is a tilemap-specific concept, and it's not clear here how it would affect the `-o` image. Can this be reworded?

Reply:
“will be ordered first”? I was considering tile IDs as also array indices (i.e. order), but I see your point.

Reply:
Sure, that works.

Reply:
How about this, then?
```suggestion
Read tiles that will be used to convert this image, will always be included in
.Fl o Aq s
output, and always get the first IDs in the tilemap emitted by
.Fl t .
```

Reply:
How about something more analogous with the `-c/--color-spec` description:

> Use the specified input tiles in addition to having `rgbgfx` automatically determine some. The input tiles will always be first in `-o` image output, and will always get the first IDs in the `-t` tilemap output.

(Not writing that in troff macros, I don't even know quite what `Aq` is.)

Reply:
Oh yeah, I like this! Let me edit your post into a suggestion so you get the credit.

(PS: pedantically, those are mandoc macros, not troff.)

Reply:
I gather that some systems use groff to render man pages, and some use mandoc? Which would explain certain rendering differences.

Reply:
It's more subtle. mdoc is a macro pack built on top of troff; and both groff and mandoc provide implementations of troff *and* of mdoc. (This is why `groff` needs to be invoked with the `-mdoc` switch.)